pymysqlreplication/row_event.py

```diff
@@ -1005,14 +1005,17 @@ def _parsed_column_charset_by_default_charset(
         column_type_detect_function,
     ):
         column_charset = []
+        position = 0
```

Comment:
@YAtOff confused..
if binlog_row_metadata = "MINIMAL" 
we can not enter this method.
because your  self.__optional_meta_data value is False

``
 def _sync_column_info(self):
        if not self.__optional_meta_data: 
            # If optional_meta_data is False Do not sync Event Time Column Schemas
            return
        if len(self.optional_metadata.column_name_list) == 0:
            # May Be Now BINLOG_ROW_METADATA = FULL But Before Action BINLOG_ROW_METADATA Mode = MINIMAL
            return
```

Reply:
Sorry, my mistake. It's `full`.
But if it's `minimal` the tests pass without asserts.